### PR TITLE
Master fix filter on blockquote bvr

### PR DESCRIPTION
**Pull request by bvr-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/static/src/scss/web_editor.common.scss [hunk 1]
```diff
@@ -725,14 +725,18 @@ blockquote {
...
+    bottom: 0;
+    right: 0;
     display: block;
     overflow: hidden;
     background-repeat: no-repeat;
     pointer-events: none;
 }
 
-section {
+section, .oe_img_bg, [data-oe-shape-data], .o_background_video, .parallax {
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/scss/web_editor.common.scss:739]: - background video is not a class of we_editor
- I made a localized fix for the Big Boxes snippet, I guess your fix is better though so you can remove my fix at the same time in this commit

File: addons/web_editor/static/src/scss/web_editor.common.scss [hunk 2]
```diff
@@ -725,14 +725,18 @@ blockquote {
     }
 }
 %o-we-background-layer {
-    @include o-position-absolute(0, 0, 0, 0);
+    position: absolute !important;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
```
Comment (qsm-odoo) [inline addons/web_editor/static/src/scss/web_editor.common.scss:732]: I merged a similar fix recently while fixing another bug so I guess you can just drop this part sorry

File: addons/web/static/lib/bootstrap/scss/_carousel.scss [hunk 3]
```diff
@@ -86,7 +86,7 @@
 
 .carousel-control-prev,
 .carousel-control-next {
-  position: absolute;
+  position: absolute !important;
```
Comment (qsm-odoo) [inline addons/web/static/lib/bootstrap/scss/_carousel.scss:89]: You are modifiying the lib... you cannot

File: addons/web/static/lib/bootstrap/css/bootstrap.css [hunk 4]
```diff
@@ -6301,7 +6301,7 @@ a.close.disabled {
 
 .carousel-control-prev,
 .carousel-control-next {
-  position: absolute;
+  position: absolute !important;
```
Comment (qsm-odoo) [inline addons/web/static/lib/bootstrap/css/bootstrap.css:6304]: You are modifying the lib... you cannot

File: addons/web/static/lib/bootstrap/scss/_carousel.scss [hunk 5]
```diff
@@ -143,7 +143,7 @@
 // slide your carousel holds.
 
 .carousel-indicators {
-  position: absolute;
+  position: absolute !important;
```
Comment (qsm-odoo) [inline addons/web/static/lib/bootstrap/scss/_carousel.scss:146]: You are modifying the lib... you cannot

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

**Comment by bvr-odoo** [comment]
@qsm-odoo I made the modifications

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
- Multiple commits in the PR, you need to mention the PR link in each commit
- Review commit message: no uppercase, wrong mentioned apps, etc

**Comment by qsm-odoo** [comment]
Does not receive CI/upgrade for some reason, let's try on another PR: https://github.com/odoo/odoo/pull/58850

**Pull request closed without merging**